KD63799/deadlinedrive
# Add create_quote functionality

Description:

This pull request adds a new feature for creating quotes. The implementation includes the following components:

CustomizeQuotes Component:

Fetches categories and quotes from the server.
Allows users to input a new quote with an author and category.
Provides success and error messages based on the response from the server.
Integrates ChooseAuthor and ChooseTheme components for selecting the author and category.
ChooseTheme Component:

Fetches categories from the server.
Allows users to search and select a category from a dropdown list.
Updates the selected category in the parent component (CustomizeQuotes).
ChooseAuthor Component:

Fetches quotes from the server and extracts unique authors.
Allows users to search and select an author from a dropdown list.
Updates the selected author in the parent component (CustomizeQuotes).
Files Changed:

src/components/CustomizeQuotes.js
src/components/ChooseTheme/ChooseTheme.js
src/components/ChooseAuthor/ChooseAuthor.js
New Dependencies:

axios for making HTTP requests.
Test Instructions:

Navigate to the Customize Quotes page.
Enter a new quote, select an author (or add a custom author), and choose a category.
Submit the form and verify that the new quote is added successfully.
Check for appropriate success and error messages.
